src/L.Routing.GraphHopper.js

```diff
@@ -54,6 +54,14 @@
 				var data;
 
 				clearTimeout(timer);
+				var fired = err ? err : resp;
```

Comment:
I'm thinking this event should only fire if the request hasn't timed out, so move it inside the conditional below. As it is now, you will get `response` events also for timed out requests, which could be confusing, although it's really an edge case.